### ISSUE TITLE
R Installation Details - Showing Old Version of R

### ISSUE THREAD

**Issue by pbond-idfg** [R-ArcGIS/r-bridge-install#56]
When I try to run the 'Install R bindings' tool, the 'Select R Version (Set As Default)' option is 3.3.1 and is greyed out and I cannot select a different version of R.  I have uninstalled 3.3.1 and installed 3.4.2 and this still happens.

When I run the 'R Installation Details' page I get:
![image](https://user-images.githubusercontent.com/6034863/32065447-0ffba6de-ba3a-11e7-8229-98b6cbcb6c28.png)

Any ideas?

P.S. I am trying to embrace ArcGIS Pro but sometimes I need to revert back to ArcMap to get something done quickly.  I did successful install the bridge in Pro, with no problems.

**Comment by scw** [comment]
Hello Pam,

I think what's happened is that the new installation didn't register itself as the default, so the older 3.3.1 listing is still in the registry. Could you try manually removing (or renaming) the C:\Program Files\R\R-3.3.1 directory, then open ArcMap and run the tool?

Cheers,
Shaun

p.s. Yes, I think most users are in a similar situation -- muscle memory wins the speed race! When I do use Pro, I find that the interface is more consistent and easier to use, but it does take some training to reach that point.

**Comment by pbond-idfg** [comment]
It doesn’t exist.

[cid:image001.png@01D34E50.51AEB270]

**Comment by pbond-idfg** [comment]
It doesn't exist.

![image](https://user-images.githubusercontent.com/6034863/32068443-9a5ee8be-ba43-11e7-9bb5-31bf17b92dd3.png)

**Comment by pbond-idfg** [comment]
Just opening the 'Install R bindings' tool I get a red X and this:
![image](https://user-images.githubusercontent.com/6034863/32072862-674a8466-ba51-11e7-9747-6a37f1b02325.png)

I ignored it (because I don't know what it means' and typed '3.4.2' in the 'Selected R Version' box. (No dropdown appeared). Got this:
![image](https://user-images.githubusercontent.com/6034863/32072923-9b1633bc-ba51-11e7-80f8-b4e393fb05b8.png)

Interesting because I don't have R bridge installed.  Maybe it is seeing the bridge I set up for ArcGIS Pro?!?  Opened the tool again and checked 'Overwrite Existing Installation' and typed in '3.4.2' and got this:
![image](https://user-images.githubusercontent.com/6034863/32072984-d097e012-ba51-11e7-85ce-fa41d032be19.png)

'R Installation Details':
![image](https://user-images.githubusercontent.com/6034863/32073300-d049c2b4-ba52-11e7-908a-a3d7d5a1065f.png)

I can use the arcgisbinding library in R studio.  So I guess it is installed?!?
![image](https://user-images.githubusercontent.com/6034863/32073342-eeb2e9a6-ba52-11e7-8281-d767df575521.png)

**Comment by pbond-idfg** [comment]
I have probably over-complicated the situation by trying to use in ArcMap and ArcPro

**Comment by wadloopoer** [comment]
Although possibly off topic, but going back to the initial question, the 'Select R Version (Set As Default)' for me (ArcMap 10.4.1) was not greyed out but was empty. I attempted to manually enter the version but this failed.  Turns out that I had a installed R @ 64-bit. Uninstalling (I did not need 64-bit) and re-installing 32-bit solved that and 'Install R bindings' then worked.

**Comment by scw** [comment]
@wadloopoer thanks for your report. Yes, because ArcMap is a 32-bit application, it can only talk to the 32-bit version of R as you found. Some libraries, like the MDB format used by Personal Geodatabases, only have 32-bit drivers and share this limitation. Microsoft has discontinued support for MDB, and there are no 64-bit versions of these libraries.

**Comment by scw** [comment]
@pbond-idfg Have you been able to resolve your issues using the bridge with ArcMap?

**Comment by andyfarmerboy** [comment]
Hi @scw I had a very similar problem to @pbond-idfg.  I have just upgraded Pro and ArcMap and the r-bridge install scripts aren't working (similar errors as described above).  However, I was able to install the bindings manually using the method described in the [screencast](https://4326.us/R/zipinst/r%20bridge%20install%20from%20zip_player.html) and I was able to get scripts working in ArcMap.  (I have run into an error in ArcGIS Pro but will post it in a new tread).
